PNA.mdk

```diff
@@ -940,10 +943,14 @@ sustain `add_entry` calls at a large fraction of their line rate, but
 it need not be at the same packet rate supported for processing
 packets that do not call `add_entry`.  The new table entry will be
 matchable when the next packet is processed that applies this table.
+Use `add_on_miss` with caution because the data plane must have
+enough information to verify that adding an entry on miss is correct.
```

Comment:
I know I have said this before, but why a special warning for add-on-miss?  Why not for every assignment statement you write?  Every action?  Every if statement?  For all of them, and everything else in a P4 program, the data plane must have enough information to verify that doing that thing is correct.  add-on-miss isn't special in that regard.